tools/openocd/stm32_h7/stm32_h7.cfg

```diff
@@ -0,0 +1,7 @@
+source [find interface/stlink-v2-1.cfg]
```

Comment:
This will not be good as we won't be using a STLink, just keep this in mind

Reply:
True we don't have a nucleo, I'll remove it